spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/TaskExplorer.java

```diff
@@ -68,5 +78,12 @@
 	 */
 	public List<TaskExecution> getTaskExecutionsByName(String taskName, int start, int count);
 
+	/**
+	 * Retrieves all the task executions within the pageable constraints.
+	 *
+	 * @param pageable the constraints for the search
+	 * @return page containing the results from the search
+	 */
```

Comment:
Any insights into what the default sorting we use is (since we need to sort by something to be reliably pagable) would be useful.